boards/BMS/Src/App/App_Accumulator.c

```diff
@@ -82,8 +93,19 @@ void App_Accumulator_RunOnTick100Hz(
             // TODO: Add broadcast BMS CAN messages when fixed pt. CAN PR has
             // been merged in
 
-            // Start cell temperature voltage conversions for the next cycle
+            // Start cell voltage conversions for the next cycle
+            accumulator->start_cell_temp_conv();
+            state = GET_CELL_TEMP_STATE;
+            break;
+
+        case GET_CELL_TEMP_STATE:
+
+            // Read cell temperatures
+            accumulator->read_cell_temperatures();
```

Comment:
Do we/should we care whether or not `read_cell_temperatures()` returns false or not? I am thinking we could check since a conversion shouldn't take longer than 10 ms (I assume we're operating at 7kHz ADC mode)

Reply:
We probably will in the future. You can refer to https://github.com/UBCFormulaElectric/Consolidated-Firmware/pull/757 and see what we would do when it returns false. For this PR i think we can leave it as is

Reply:
We can also run this off a 10Hz task. We really dont need to be collecting data at 100Hz rn

Reply:
it's fine at 100 Hz

Reply:
its slightly riskier as we may trip the watchdog. unlikely, but more likely when we run it in the 100hz task